tests/lib/path_utils.py

```diff
@@ -20,3 +22,29 @@ def expand_env_vars(obj):
     else:
         obj = os.path.expandvars(obj)
     return obj
+
+
+def get_os_name():
```

Comment:
I'd prefer to reuse already added functionality from https://github.com/openvinotoolkit/openvino/blob/master/tests/time_tests/test_runner/utils.py. To do so, need to move it to a common utilities.

Also it's based on `platform.system` with which @asomsiko was argued about. But I see no issues here at least because or CI is based on such functionality

Reply:
I support reusing as a library, no matter how it detects OS

Reply:
Will be done in the next PR